README.md

```diff
@@ -26,18 +26,20 @@ these types with a simple gzip.
 
 - File patterns supported
 
-  - *.fastq$
-  - *.txt$
+  - `*.csv`
+  - `*.fastq`
+  - `*.txt`
 
 #### Archiving files
 
 - Files patterns supported
 
-  - *.csv$
-  - *.fast5$
-  - *.gz$
-  - *.md$
-  - *.pdf$
+  - `*.csv`
```

Comment:
Should this be removed, since `*.csv` is only copied if compressed (valet/predicates.go#L69)?

Reply:
I think you're right that the lists aren't consistent - I intended the two lists to be "files that will be compressed" and "files that are supported", so really I need to add `fastq` and `txt` to the second list.